tests/installation/bootloader_svirt.pm

```diff
@@ -345,7 +345,7 @@ sub run {
 
     # connects to a guest VNC session
     select_console('sut', await_console => 0);
-    vmware_set_permanent_boot_device($boot_device);
+    vmware_set_permanent_boot_device($boot_device) unless get_var('UEFI');
```

Comment:
I suggest to add a new function in lib/Utils/Backend.pm:

```
sub is_uefi_boot {

    if (check_var('UEFI', '1') or check_var('IPXE_UEFI', '1')) {
        return 1;
    }
    return 0;
}
```

Then call it here.

Reply:
Updated. Thanks

Reply:
Could you help move is_uefi_boot to lib/Utils/Backend.pm ?